cvmfs/swissknife_ingest.h

```diff
@@ -38,6 +38,9 @@ class Ingest : public Command {
     r.push_back(Parameter::Optional(
         'C', "create a new catalog where the tar file is extracted"));
 
+    r.push_back(Parameter::Optional('P', "session_token_file"));
```

Comment:
How are these two parameters actually used?

Reply:
Here: https://github.com/cvmfs/cvmfs/blob/74efe86614d6e145eb75e93d145adfd6865a4035/cvmfs/swissknife_ingest.cc#L76-L90

You don't think is necessary?

Reply:
I understand now, thanks.